src/DefinitionsReader.php

```diff
@@ -504,6 +504,10 @@ public function readFile($file)
      */
     private function getDOM($file)
     {
+        if (!file_exists($file)) {
```

Comment:
`$file` can be also in the form `http://...`. Does `file_exists` supports remote files?

Reply:
It doesn't...

file_get_contents could be used. But it could return false also for 403 statusCode. 
So it's more like "couldn't load the file" error anyway.

```php
    private function getDOM($file)
    {
        $content = file_get_contents($file);

        if (false === $content) {
            throw new IOException("Invalid file path or url $file");
        }

        $xml = new DOMDocument('1.0', 'UTF-8');
        if (!$xml->load($content, LIBXML_NONET)) {
            print_r(libxml_get_errors());
            die();
            throw new IOException("Can't load the file $file");

        }
        return $xml;
    }
```

So could die; be removed? It at least will tell which wsdl is problem in https://github.com/goetas-webservices/wsdl-reader/blob/master/src/DefinitionsReader.php#L510

Reply:
i think you can just remove ` print_r(libxml_get_errors());            die();` that is there by mistake. and things will work